espanol/poeta/poeta.md

```diff
@@ -0,0 +1 @@
+Y al observar a su alrededor, el intrigado hombre penso que los hechos ocurridos darian fruto a una hermosa historia.  Pero, para recodar a su hermana, insistió en que tenia que ser un poema.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.